pkg/mounter/ossfs_test.go

```diff
@@ -111,7 +114,44 @@ func Test_getPasswdSecretVolume(t *testing.T) {
 	}
 }
 
+const (
+	masterUrl = "https://api-server-host:6443"
+
+	mockConfigMapJson = `
+{
+	"apiVersion": "v1",
+	"kind": "ConfigMap",
+	"metadata": {
+		"name": "ossfs-mime-config"
+	},
+	"data": {
+		"volumeId": "application/json json"
+	}
+}`
+)
+
+func prepareFakeK8sContext() {
+	options.MasterURL = masterUrl
+}
+
+func registerConfigMapResponder(exists bool) {
+	url := fmt.Sprintf("=~^%s/api/v1/namespaces/%s/configmaps/.*\\z", masterUrl, FusePodNamespace)
+	if !exists {
+		httpmock.RegisterResponder("GET", url, httpmock.NewStringResponder(404, ""))
+		return
+	}
+	responder := httpmock.NewStringResponder(200, mockConfigMapJson)
+	responder = responder.HeaderSet(map[string][]string{
+		"Content-Type": {"application/json"},
+	})
+	httpmock.RegisterResponder("GET", url, responder)
+}
+
 func Test_AddDefaultMountOptions(t *testing.T) {
+	httpmock.Activate()
```

Comment:
Please try to inject a fake clientset into the tested instance, and avoid modifying global state, to enable parallel test in the future.
And, why not use the fake clientset implemented in client-go?